x-pack/legacy/plugins/lens/public/app_plugin/app.test.tsx

```diff
@@ -334,65 +334,6 @@ describe('Lens App', () => {
 
         expect(getButton(instance).disableButton).toEqual(false);
       });
-
-      it('saves the latest doc and then prevents more saving', async () => {
-        const args = makeDefaultArgs();
-        args.editorFrame = frame;
-        (args.docStorage.save as jest.Mock).mockResolvedValue({ id: '1234' });
-
-        const instance = mount(<App {...args} />);
-
-        expect(frame.mount).toHaveBeenCalledTimes(1);
-
-        const onChange = frame.mount.mock.calls[0][1].onChange;
-        onChange({ filterableIndexPatterns: [], doc: ({ id: undefined } as unknown) as Document });
-
-        instance.update();
-
-        expect(getButton(instance).disableButton).toEqual(false);
-
-        act(() => {
-          getButton(instance).run(instance.getDOMNode());
-        });
```

Comment:
If I recall correctly this isn't the only test that runs the `save` behavior, but since it's changed in this PR my suggestion is to grab `instance.find(SavedObjectSaveModal).prop('onSave')({ ... })` and call it instead of via the DOM